data/neerajkumar161.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "Neeraj Kumar",
+  "type": "personal",
+  "bio": "Backend Developer | **Node.js** | TypeScript Lover",
+  "avatar": "",
```

Comment:
Currently, you don't need to add an ` avatar`. Delete this line.

Reply:
```suggestion
```